### ISSUE TITLE
host a GitHub tutorial for interested ambassadors

### ISSUE THREAD

**Issue by semioticrobotic** [open-organization-ambassadors/ambassador-board-room#10]
I'd like to follow up on the group's desire to participate in a small GitHub "training" session we can use to explore ways we might better leverage the platform as a collaboration tool. Ambassador @whit537 has agreed to assist. I'll float the following outline. Thoughts, anyone?

# A GitHub Introduction
## Basics
- What is an issue tracker?
- What is GitHub?
    - Anatomy of the user's home page
    - Anatomy of a repository landing page
- Basic vocabulary/nomenclature
    - Repository
    - Issue
    - Clone
    - Push/pull
    - Pull request
    - Watch/Star
## Key Ambassador Skills
- Common notifications and actions
- Opening and/or commenting on an issue
- Markdown syntax
- Referencing issues and people
- Interacting with GitHub via email
## Advanced participation
- Team permissions
- Cloning repositories
- Making pull requests
- Creating milestones
- Creating projects
- Keeping local, forked branches in sync

**Comment by semioticrobotic** [comment]
Additional notes:

- When we've cemented a plan, I will schedule the tutorial on a Thursday at 10:00 Eastern, since we already know that time typically works for the broadest swatch of the ambassadors. We'll meet via BlueJeans as we normally do.
- We'll keep the session capped at one hour in length

**Comment by whit537** [comment]
Outline looks good, @semioticrobotic. What about recording the session and posting the video? That way we could use it as a resource when onboarding future ambassadors, or simply for those current ambassadors unable to make the call.

**Comment by semioticrobotic** [comment]
Yes, for sure, @whit537. I'd also like to "reverse outline" the presentation that ultimately emerges from this and make a single-page guide to that video (easier to follow along or skip ahead). What do you recommend for making this kind of recording?

**Comment by whit537** [comment]
I've generally had good luck with Google Hangouts. They may have merged the product into YouTube since I used it last, but last I checked it was a great way to both live-stream and automatically archive to YouTube for future reference. Summarizing a longer call is a great best practice for openness. Dumping an hour of video on the web is of limited utility because it's so difficult to find the needle in the haystack, whereas an outline that includes timestamps (YouTube makes this easy within video description and comments) helps make the content much more accessible. Good thinking!

**Comment by semioticrobotic** [comment]
Ok then. I will set something up, float a calendar invitation, and then update that invitation with the appropriate linkage for whatever tool or channel we're using.

**Comment by semioticrobotic** [comment]
I finally have a little extra bandwidth for planning  this more specifically, @whit537. How would you like to go about running the workshop? I'm very open to suggestions and ideas.

**Comment by whit537** [comment]
Are you comfortable driving the session, @semioticrobotic? I would love for you to take that role, and I can be available to chime in as appropriate. From what I've seen of your work with GitHub so far, you've got a much better handle on the platform than you let on! :-)

**Comment by semioticrobotic** [comment]
I can certainly kick us off and provide the frame/overview, but I may need to toss to you on some of the finer points because I have a feeling you'll be able to better explain them. When I have a more concrete outline, I'll let you know where I might need the most help.

**Comment by whit537** [comment]
Sounds perfect, @semioticrobotic. Let me know what parts you'd like me to cover!

**Comment by semioticrobotic** [comment]
I'm fleshing out a personal presentation outline, @whit537, and it occurs to me that the first part of the workshop ("What s an Issue Tracker?") is in your wheelhouse (given #9). Since that article will not be published before the workshop, can you summarize a few key takeaways for everyone before we delve into GitHub itself?

**Comment by semioticrobotic** [comment]
We'll get work right away posting the recording online!

**Comment by whit537** [comment]
NOOOOoooooooooo..............!

Gah! I am SO SORRY I totally spaced this! No excuse, just got caught up in other work and lost track of the morning. :-(

That said, I'm quite curious to watch the video and see how it went! How did @semioticrobotic in the driver's seat with no copilot?

**Comment by semioticrobotic** [comment]
Well, @whit537, you can [see for yourself](https://www.youtube.com/watch?v=hoWOJHYxMqI). I think it's pretty clear that I'm better with a co-pilot. Ha!

**Comment by whit537** [comment]
I'm sure you did great. :-) I've downloaded the video to watch on the bus tomorrow, looking forward to it! 📺  🚌  💃 😆

**Comment by whit537** [comment]
You did a truly wonderful job, @semioticrobotic. This is the perfect intro to GitHub for those who may be scared to jump in—covering all of the major features of the platform in an accessible way. Good work! So happy to have this to share with people!

**Comment by semioticrobotic** [comment]
Wow! Thanks, @whit537! That means a lot coming from you. In a year or
so, we might consider co-recording v2.0 to streamline the demonstration
and incorporate more of the group's oft-used features.

**Comment by semioticrobotic** [comment]
Wow! Thanks, @whit537! That means a lot coming from you. In a year or so, we might consider co-recording v2.0 to streamline the demonstration and incorporate more of the group's oft-used features.

**Comment by whit537** [comment]
Hey @semioticrobotic, talking with some Gratipay crew here and we realized that clones and forks *are* basically the same—one is a copy between GitHub and your local computer (clone), while the other is a copy that stays within GitHub (fork). You weren't so far wrong after all! :-)

P.S. Ready to close this ticket? What is left to do here?

**Comment by semioticrobotic** [comment]
I like the way your crew thinks, @whit537—especially when it makes me sound more correct. Thanks for that!

Yep. Let's close it up. Thanks for all the input and participation, everyone!
